docs/aws-sdk-for-iOS-with-minio.md

```diff
@@ -0,0 +1,58 @@
+# How to use AWS SDK for iOS(swift-3) with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+In this recipe we will learn how to use `AWSS3` for iOS with Minio server. `AWSS3` is the official AWS S3 SDK for the swift/objective-c programming language.
+
+## 1. Prerequisites
+
+Install Minio Server from [here](http://docs.minio.io/docs/minio).
+
+## 2. Installation
+
+Setup `AWSS3` for iOS from the official AWS iOS SDK docs [here](http://docs.aws.amazon.com/mobile/sdkforios/developerguide/setup-aws-sdk-for-ios.html)
+
+we only need 'AWSS3'
+
+## 3. Example
+
+please replace `accessKey`, `secretKey`, and `url`, change the region base on what you need, service must set to `.S3`
+
+(awss3 will auto remove port number if you type `xxxx:9000` in `url`, currently it only support full url without port, so please make sure you have a domain map to port 9000, you may need refer to this [Setup Nginx proxy with Minio Server](./docs/setup-nginx-proxy-with-minio.md))
+
```

Comment:
Please provide the Minio docs link instead of GitHub doc -- https://docs.minio.io/docs/setup-nginx-proxy-with-minio